assessments/guide/security-reviewer.md

```diff
@@ -0,0 +1,31 @@
+# Security Reviewers
+
+Reviewers will try to understand the
+system and probe its security.  Specifically design level issues are meant
+to be addressed as well as high level problems with the project’s setup and
+operation. This is meant to provide an independent analysis and estimation
+of the above information.  The goal is to ask questions to understand if
+there are hidden assumptions, underestimated risk, or design issues that
+harm the security of the project.  It may be useful to reach out to
+community members to understand the answers to some questions, especially
+involving deployment scenarios and the impact of attacks.
+
+## Qualifications
+
+Unless approved by the SIG chairs, at least one of the reviewers will
+have previously performed a SAFE WG audit.  (Exemptions are expected to be
+granted bootstrap the process but only in extreme cases later on.)  In
+general, a reviewer should have performed security audits for diverse
+organizations.  The ideal reviewer should also have been the recipient
+of security audits for a software project they manage.  Note that it is
+encouraged to have participation (shadowing) from participants that are not
+yet qualified to help them gain the necessary skills to be a reviewer
```

Comment:
Perhaps we should explicitly catalog what those skills are?  Not only is that useful for those hoping to participate, but it lends itself to segregating the work and parallel processing.  And not all experts will be experts in everything. for example I have deep crypto review expertise, but very little linux kernel review expertise.

Reply:
There's not a clear consensus on what this is, so we decided to do a number of assessments before adding more detail -- which i think we set as 6-8 assessments, but could revisit earlier if it causes friction in the process or there are specific recommendations that help the process

@rficcaglia -- thanks for chiming in on https://github.com/cncf/sig-security/issues/142 -- feel free to add more thoughts / ideas there on this topic, and we'll revisit at a future checkpoint